controllers/users.js

```diff
@@ -462,6 +464,27 @@ const filterUsers = async (req, res) => {
   }
 };
 
+const migrate = async (req, res) => {
+  try {
+    // Fetch user data from GitHub API for each document in the users collection
+    const usersSnapshot = await firestore.collection("users").get();
+    for (const userDoc of usersSnapshot.docs) {
+      const username = userDoc.data().github_id;
```

Comment:
can we call it `githubId` instead of `username` ?

Reply:
Okay I'll make the changes